core/optaplanner-core-impl/src/test/java/org/optaplanner/core/impl/domain/lookup/LookUpStrategyIdOrFailTest.java

```diff
@@ -15,14 +15,14 @@
 import org.optaplanner.core.impl.testdata.domain.clone.lookup.TestdataObjectNoId;
 import org.optaplanner.core.impl.testdata.domain.clone.lookup.TestdataObjectPrimitiveIntId;
 
-class LookUpStrategyIdOrFailTest {
+class LookUpStrategyIdOrFailTest extends AbstractLookupTest {
 
     private LookUpManager lookUpManager;
 
     @BeforeEach
     void setUpLookUpManager() {
         lookUpManager = new LookUpManager(
-                new LookUpStrategyResolver(DomainAccessType.REFLECTION, LookUpStrategyType.PLANNING_ID_OR_FAIL_FAST));
+                createLookupStrategyResolver(DomainAccessType.REFLECTION, LookUpStrategyType.PLANNING_ID_OR_FAIL_FAST));
     }
```

Comment:
It appears to me that `lookupManager` could be part of the parent, filled there by calling an abstract method that the child would implement.

Otherwise I see no point for `AbstractLookupTest`.

Reply:
Good catch, I thought some of the tests don't use the `lookupManager`, but that's not true.